src/main/java/redis/clients/jedis/commands/BasicCommands.java

```diff
@@ -3,6 +3,9 @@
 import redis.clients.jedis.DebugParams;
 
 public interface BasicCommands {
+  
+  RedisPipeline pipelined();
```

Comment:
This interface contains only actual Redis commands. We should keep it this way.

Reply:
:+1: